src/core/output-layout.cpp

```diff
@@ -581,6 +586,29 @@ struct output_layout_output_t
             wlr_output_set_custom_mode(handle, mode.width, mode.height, mode.refresh);
         }
 
+        if (!is_nested_compositor)
+        {
+            if ((handle->adaptive_sync_status == WLR_OUTPUT_ADAPTIVE_SYNC_DISABLED) && current_state.vrr)
+            {
+                wlr_output_enable_adaptive_sync(handle, current_state.vrr);
+                wlr_output_commit(handle);
+                if (handle->adaptive_sync_status == WLR_OUTPUT_ADAPTIVE_SYNC_DISABLED)
+                {
+                    LOGE("Failed to enable adaptive sync on output: ", handle->name);
+                    wlr_output_enable_adaptive_sync(handle, false);
+                } else
+                {
+                    LOGI("Enabled adaptive sync on output: ", handle->name);
```

Comment:
Sounds like a good candidate for LOGD, not LOGI. Optionally, maybe even add a new logging category to debugging.hpp (output-related)?

Reply:
Sure, we can make it LOGD but I'm not sure it warrants a new debug category.

By the way, is there a way to enable all debug categories at once with a single option?